### PR TITLE
Update queens from upstream

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

File: docker/rabbitmq/Dockerfile.j2 [hunk 1]
```diff
@@ -27,7 +27,7 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
          ] %}
     {% else %}
          {% set rabbitmq_packages = rabbitmq_packages + [
-             'http://www.rabbitmq.com/releases/rabbitmq-server/v3.6.5/rabbitmq-server_3.6.5-1_all.deb',
+             'https://www.rabbitmq.com/releases/rabbitmq-server/v3.6.5/rabbitmq-server_3.6.5-1_all.deb',
```
Comment (oneswig) [inline docker/rabbitmq/Dockerfile.j2]: This looks potentially significant.

File: tools/setup_RedHat.sh [hunk 2]
```diff
@@ -63,14 +15,12 @@ sudo yum -y install libffi-devel openssl-devel docker-ce btrfs-progs
...
 
-setup_disk
-
 # Setup Docker
 sudo mkdir /etc/systemd/system/docker.service.d
 sudo tee /etc/systemd/system/docker.service.d/kolla.conf << EOF
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd --storage-driver btrfs --insecure-registry=0.0.0.0/0
+ExecStart=/usr/bin/dockerd --storage-driver overlay2 --insecure-registry=0.0.0.0/0
```
Comment (oneswig) [inline tools/setup_RedHat.sh]: This looks quite significant!

Comment (markgoddard) [reply tools/setup_RedHat.sh]: It's just a script for setting up CI nodes.

File: docker/rabbitmq/Dockerfile.j2 (hunk 1 above)
Comment (markgoddard) [reply docker/rabbitmq/Dockerfile.j2]: This is the fix we want to bring in for SciCORE.

Comment (markgoddard) [reply docker/rabbitmq/Dockerfile.j2]: Well, the related line below is actually what we need, but close enough.

**Pull request merged**